vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -1154,4 +1154,11 @@
 - name: configure Prc-Dockets, if needed
   include_tasks: configure_prc_dockets.yml
 
-
+- name: add pdfTron license if defined
+  become: yes
+  become_user: arkcase
+  lineinfile:
```

Comment:
This will update the file every time, since the line includes the regexp. It is not idempotent, which we generally shoot for.  But at least it doesn't add onto the line... the file will be updated with the same text every time.

Reply:
Dave, thanks for the comment. I update the code accordingly.